pyproject.toml

```diff
@@ -21,13 +21,16 @@ classifiers = [
 ]
 
 dependencies = [
-    "greenlet",
+    "greenlet >= 1",
     "typing-extensions >= 4.6.0",
 ]
 
 [tool.setuptools.packages]
 find = {}  # Scanning implicit namespaces is active by default
 
+[tool.setuptools.dynamic]
+version = {attr = "awaitlet.__version__"}
```

Comment:
**Michael Bayer** (zzzeek) wrote:

oh

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/awaitlet/+/5336